src/mpyl/steps/build/docker_build.py

```diff
@@ -75,7 +75,11 @@ def execute(self, step_input: Input) -> Output:
             contents = "\n".join(DOCKER_IGNORE_DEFAULT)
             ignore_file.write(contents)
 
-        build_args: dict[str, str] = {}
+        build_args: dict[str, str] = {
```

Comment:
I think we should put this definition somewhere central so it's not defined twice